superset-frontend/src/views/CRUD/annotation/AnnotationList.tsx

```diff
@@ -186,6 +243,24 @@ function AnnotationList({ addDangerToast }: AnnotationListProps) {
     hasHistory = false;
   }
 
+  const EmptyStateButton = (
+    <Button
+      buttonStyle="primary"
+      onClick={() => {
+        handleAnnotationEdit(null);
+      }}
+    >
+      <>
+        <i className="fa fa-plus" /> {t('Annotation Layer')}
+      </>
+    </Button>
+  );
+
+  const emptyState = {
+    message: 'No annotation yet',
```

Comment:
`No annotations yet`, also we should be wrapping this in `t(...)`

Just realized we aren't doing that for the message on `AnnotationLayersList`, any chance we can get that updated as well?